frontend/test/pytest/python_interface/transforms/quantum/test_xdsl_parity_synth.py

```diff
@@ -479,7 +478,7 @@ def circuit(x: float, y: float, z: float):
             return qml.state()
 
         raw_circuit = qml.qjit(circuit)
-        compiled_circuit = qml.qjit(parity_synth_pass(circuit), target="mlir")
+        compiled_circuit = qml.qjit(parity_synth_pass(circuit))
```

Comment:
This change was made because there was a jit-compilation warning being raised because we were trying to execute a circuit that had not been compiled all the way and required recompilation.